gitversion.yml

```diff
@@ -1,14 +1,14 @@
 assembly-versioning-scheme: MajorMinorPatch
-mode: Mainline
-next-version: 2.0.0
+mode: ContinuousDeployment
+next-version: 2.1
```

Comment:
Do we bump to 2.1 ? we didn't release 2.0 stable yet.

Reply:
Good point ... fixed!